source/includes/steps-local-keyfile.yaml

```diff
@@ -0,0 +1,101 @@
+---
+title: "Generate an Encryption Key."
+ref: generate-local-key
+level: 4
+content: |
+
+  To configure client-side field level encryption for a locally managed
+  key, you must specify a base64-encoded 96-byte string with no line
+  breaks. Run the following command on the command line (*not* within
+  ``mongosh``) to generate a key matching these requirements:
+
+  .. code-block:: javascript
+
+     echo $(head -c 96 /dev/urandom | base64 | tr -d '\n')
```

Comment:
It might be easier to tell people to run `crypto.randomBytes(96).toString('base64')` *inside* the shell, mongosh supports that (the old shell wouldn't) and it's cross-platform

Reply:
Thanks! This is a great tip!